tests/Fixtures/Kernel.php

```diff
@@ -43,7 +43,7 @@ public function registerBundles(): iterable
             yield new DAMADoctrineTestBundle();
         }
 
-        if ('migrate' === \getenv('FOUNDRY_RESET_MODE')) {
+        if (\getenv('USE_MIGRATIONS')) {
```

Comment:
I changed the name of the env var in order to remove deprecations triggers in the tests